src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -38,12 +40,15 @@ public StatefulPDFMerger(Map<BundleDocument, File> documents, Bundle bundle) {
         }
 
         public File merge() throws IOException {
+            pdfOutline.addBundleItem(bundle.getTitle());
             if (bundle.hasTableOfContents()) {
                 this.tableOfContents = new TableOfContents(document, bundle);
                 currentPageNumber += tableOfContents.getNumberPages();
+                pdfOutline.addItem(0, "Index Page");
             }
 
             addContainer(bundle);
+            pdfOutline.setBundleItemDest(0);
```

Comment:
I'm not sure I like this method name - it doesn't quite tell me what's going on. Maybe `setRootOutlineLinkToPage` ... maybe that's worse :D up to you!